src/shared/client/cg_api.cpp

```diff
@@ -315,6 +315,17 @@ void trap_R_Add2dPolysIndexedToScene( polyVert_t* polys, int numPolys, int* inde
 	cmdBuffer.SendMsg<Render::Add2dPolysIndexedMsg>(mypolys, numPolys, myindices, numIndexes, trans_x, trans_y, shader);
 }
 
+void trap_R_PushMatrix( const matrix_t matrix )
```

Comment:
Can we get some comments about what it's for? Does it only work for 2D rendering?

Reply:
I added them in the renderer rather than in the trap calls to make the semantics clear.